tests/raftstore/test_multi.rs

```diff
@@ -121,6 +125,49 @@ fn test_multi_lost_majority<T: Simulator>(cluster: &mut Cluster<T>, count: usize
 
 }
 
+fn test_multi_random_restart<T: Simulator>(cluster: &mut Cluster<T>,
+                                           node_count: usize,
+                                           restart_count: u32) {
+    cluster.bootstrap_region().expect("");
+    cluster.start();
+
+    let (key, value) = (b"a1", b"v1");
+
+    assert_eq!(cluster.get(key), None);
+    cluster.put(key, value);
+    assert_eq!(cluster.get(key), Some(value.to_vec()));
+
+    let mut rng = rand::thread_rng();
+    for _ in 1..restart_count {
+        let id = 1 + rng.gen_range(0, node_count as u64);
+        cluster.stop_node(id);
+        cluster.run_node(id);
+        wait_until_node_online(cluster, id);
+        assert_eq!(cluster.get(key), Some(value.to_vec()));
```

Comment:
write a new value here and get it. 
use must_get_equal is ok, no need using cluster get.